src/mlx/warnings/code_quality.py

```diff
@@ -0,0 +1,136 @@
+
+import hashlib
+from pathlib import Path
+
+
+class Finding:
+    """Code quality violation"""
+
+    """A dictionary mapping fingerprints (unique hashes) to instances of Finding"""
+    fingerprints = {}
+
+    _severity = ""
+    _path = ""
+    _description = ""
```

Comment:
I think that description doesn't need a property and setter, so no need for a protected counterpart then.

Reply:
Resolved in 2043fc123c1c23906a2242ad83aa55aed651ee3d